config/daemon/index.md

```diff
@@ -85,6 +108,35 @@ documentation. Some places to go next include:
 - [Configure storage drivers](/engine/userguide/storagedriver/index.md)
 - [Container security](/engine/security/index.md)
 
+## Docker daemon directory
+
+The Docker daemon persists all data in a single directory. This tracks everything
+related to Docker, including containers, images, volumes, service definition,
+and secrets.
+
+By default this directory is:
+
+* `/var/lib/docker` on Linux.
+* `C:\ProgramData\docker` on Windows.
+
+You can configure the Docker daemon to use a different directory, using the
+`data-root` configuration option.
+
+Since all the state of a Docker daemon is kept on this directory, make sure
+you use a dedicated directory for each daemon. If two daemons share the same
+directory, as an example using an NFS share, you might get errors that are
+difficult to troubleshoot.
+
```

Comment:
If two daemons share the same directory, for example, an NFS share, you might ...